R/install-bitbucket.r

```diff
@@ -3,36 +3,42 @@
 #' This function is vectorised so you can install multiple packages in
 #' a single command.
 #'
+#' To install from a private repo, or more generally, access the Bitbucket API
+#' with your own credentials, you will need to get an access token. You can
+#' create an access token following the instructions found in the
+#' \href{https://confluence.atlassian.com/bitbucket/app-passwords-828781300.html}{Bitbucket
+#' App Passwords documentation}. This PAT requires read-only access to your
+#' repositories and pull requests. Then store your Bitbucket user name and PAT
+#' separated by a colon in the environment variable \code{BITBUCKET_AUTH} (e.g.
```

Comment:
I think we should be consistent with the Github convention and call the environment variable BITBUCKET_PAT and function `bitbucket_pat()`